common/views/components/InstallationPromo/InstallationPromo.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import {spacing, font} from '../../../utils/classnames';
+
+type Props = {|
+  id: string,
+  title: string
+|}
+
+const InstallationPromo = ({ id, title }: Props) => (
+  <a
+    data-component='InstallationPromo'
+    data-track-event={JSON.stringify({
+      category: 'component',
+      action: 'InstallationPromo:click'
+    })}
+    id={id}
+    href={`/installations/{id}`}
+    className='plain-link promo-link bg-cream rounded-top rounded-bottom overflow-hidden flex flex--column'>
```

Comment:
`rounded-corners` does the job of both `rounded-top` and `rounded-bottom` and saves you 10 keystrokes!

Reply:
👍 saved 10 keystrokes by spending keystrokes.
Ah, capitalism.